sphinx_book_theme/theme.conf

```diff
@@ -7,14 +7,16 @@ stylesheet = sphinx-book-theme.27151f2de8a6da6e72b57ec231ceb131.css
 
 [options]
 single_page = False
-expand_toc_sections = []  # DEPRECATE after a few release cycles
+# DEPRECATE after a few release cycles
```

Comment:
As mentioned at the top of the file, you shouldn’t directly write to this file; it it created in the pre-commit from src/jinja/theme.conf.j2

Reply:
hmm I thought I had edited the `.j2` file, the diff shows up here: https://github.com/executablebooks/sphinx-book-theme/pull/363/files#diff-f3d11a1ed4be749cbb3f797cd92269c7b9f3fcee90efce84f66c4bfb052dfc5bR10

did I do that wrong?

Reply:
Ah yeh your write, I was accidentally only looking at the pre-commit commit change